services/service.go

```diff
@@ -8,13 +8,19 @@ import (
 // State of the service. See Service interface for full state diagram.
 type State int
 
+// New: Service is new, not running yet. Initial state.
```

Comment:
Isn't it better to move each description to its corresponding `const`? That should cause each `const` to be documented I would think, whereas I wonder if this whole block now gets associated with `const New`?

Reply:
> Isn't it better to move each description to its corresponding const?

Yes.